Extension/src/LanguageServer/client.ts

```diff
@@ -2458,6 +2461,7 @@ export class DefaultClient implements Client {
             const compilerPathAndArgs: util.CompilerPathAndArgs =
                 util.extractCompilerPathAndArgs(!!settings.legacyCompilerArgsBehavior, c.compilerPath, c.compilerArgs);
             modifiedConfig.compilerPath = compilerPathAndArgs.compilerPath;
+            this.model.currentCompiler.setValueIfActive(modifiedConfig.compilerPath ?? "");
```

Comment:
This shouldn't be set for each configuration, right?

Reply:
That was from a previous implementation. I'll remove it